app/models/concerns/basic_geo_metadata.rb

```diff
@@ -3,15 +3,63 @@ module BasicGeoMetadata
   extend ActiveSupport::Concern
 
   included do
-    # Defines the GeoRSS bounding box for the resource
-    # From the spec: A bounding box is a rectangular region, often used to define the extents of a map
-    # or a rough area of interest. A box contains two space seperate latitude-longitude pairs, with each
-    # pair separated by whitespace. The first pair is the lower corner, the second is the upper corner.
-    #
-    # @see http://www.georss.org/simple.html GeoRSS Simple Specification
+    # The following properties are inherited from Curation Concerns' basic metadata
```

Comment:
for the sake of completeness, should we also mention the properties inherited from Curation Concerns' required metadata?